tests/features/java/java_s2i.feature

```diff
@@ -171,3 +171,11 @@ Feature: Openshift OpenJDK S2I tests
       Given s2i build https://github.com/jboss-container-images/openjdk from tests/OPENJDK-1549 with env using ubi9
        | variable           | value    |
        | MAVEN_ARGS         | validate |
+
+  Scenario: Ensure that run-env.sh placed in the JAVA_APP_DIR is sourced in the run script before launching java
+    Given s2i build https://github.com/jhuttana/openjdk-test-applications from quarkus-quickstarts/getting-started-3.0.1.Final-nos2i using run_env_sh_test_coverage1
+    | variable            | value        |
+    | S2I_ENABLE_JLINK    | true         |
+    | S2I_SOURCE_DATA_DIR | ./           |
+    | S2I_TARGET_DATA_DIR | /deployments |
+    Then container log should contain INFO exec -a "someUniqueString" java  -cp "." -jar
```

Comment:
This is what I see if I try running the s2i-output container manually:

```log
INFO exec -a "someUniqueString" java -XX:MaxRAMPercentage=80.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+ExitOnOutOfMemoryError -cp "." -jar /deployments/quarkus-app/quarkus-run.jar
```

The test is failing because the JVM tuning parameters (XX:MaxRAMPercentage=80.0  etc.) aren't in the match string. I suggest trimming the line down to stop at "java", i.e., don't try to match `-cp "." -jar` (or any JVM tuning parameters which this test is  not concerned about, and which might change)